build/common.sh

```diff
@@ -85,22 +86,26 @@ readonly KUBE_RSYNC_PORT="${KUBE_RSYNC_PORT:-}"
 # mapped to KUBE_RSYNC_PORT via docker networking.
 readonly KUBE_CONTAINER_RSYNC_PORT=8730
 
+# These are the base images for the Docker-wrapped binaries.
+readonly KUBE_GORUNNER_IMAGE="${KUBE_GORUNNER_IMAGE:-$KUBE_BASE_IMAGE_REGISTRY/go-runner:buster-v2.3.1}"
+readonly KUBE_DEBIAN_IPTABLES_IMAGE="${KUBE_DEBIAN_IPTABLES_IMAGE:-$KUBE_BASE_IMAGE_REGISTRY/debian-iptables:buster-v1.5.0}"
+
+# This is the image used in a multi-stage build to apply capabilities to Docker-wrapped binaries.
+readonly KUBE_BUILD_SETCAP_IMAGE="${KUBE_BUILD_SETCAP_IMAGE:-$KUBE_BUILD_SETCAP_REGISTRY/setcap:buster-v1.4.0}"
```

Comment:
Why isn't this defaulted to KUBE_BASE_IMAGE_REGISTRY/setcap:buster-v1.4.0?
Perhaps it would make sense to add KUBE_BUILD_GORUNNER_REGISTRY and KUBE_BUILD_DEBIAN_IPTABLES_REGISTRY and have all 3 defaulted to KUBE_BUILD_IMAGE_REGISTRY?

Reply:
TAL at the updated commit